etc/profile-a-l/cachy-browser.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for Cachy-Browser
+# Description: Librewolf fork based on enhanced privacy with gentoo patchset
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cachy-browser.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/cachy
+noblacklist ${HOME}/.cachy
+
+mkdir ${HOME}/.cache/cachy
+mkdir ${HOME}/.cachy
+whitelist ${HOME}/.cache/cachy
+whitelist ${HOME}/.cachy
+
+# Add the next lines to your cachy-browser.local if you want to use the migration wizard.
+#noblacklist ${HOME}/.mozilla
+#whitelist ${HOME}/.mozilla
+
+# To enable KeePassXC Plugin add one of the following lines to your cachy-browser.local.
+# NOTE: start KeePassXC before CachyBrowser and keep it open to allow communication between them.
+#whitelist ${RUNUSER}/kpxc_server
+#whitelist ${RUNUSER}/org.keepassxc.KeePassXC.BrowserServer
+
+whitelist /usr/share/doc
+whitelist /usr/share/gtk-doc/html
+whitelist /usr/share/mozilla
+whitelist /usr/share/webext
+include whitelist-usr-share-common.inc
+
+# Add the next line to your cachy-browser.local to enable private-bin (Arch Linux).
+#private-bin dbus-launch,dbus-send,cachy-browser,sh
+# Add the next line to your cachy-browser.local to enable private-etc.
+# NOTE: private-etc must first be enabled in firefox-common.local.
+#private-etc cachy-browser
+
+dbus-user filter
+# Add the next line to your cachy-browser.local to enable native notifications.
+#dbus-user.talk org.freedesktop.Notifications
+# Add the next line to your cachy-browser.local to allow inhibiting screensavers.
+#dbus-user.talk org.freedesktop.ScreenSaver
+# Add the next lines to your cachy-browser.local for plasma browser integration.
+#dbus-user.own org.mpris.MediaPlayer2.plasma-browser-integration
+#dbus-user.talk org.kde.JobViewServer
+#dbus-user.talk org.kde.kuiserver
+# Add the next three lines to your cachy-browser.local to allow screensharing under Wayland.
+#whitelist ${RUNUSER}/pipewire-0
+#whitelist /usr/share/pipewire/client.conf
+#dbus-user.talk org.freedesktop.portal.*
+# Also add the next line to your cachy-browser.local if screensharing does not work with
+# the above lines (depends on the portal implementation).
+#ignore noroot
+ignore dbus-user none
```

Comment:
Should be based on master librewolf.profile.

Reply:
it is actually
or you mean to include librewolf.profile instead of firefox
`include librewolf.profile`

btw will it lead to any problem for example with cachy-browser{-beta,-nightly,etc}?

Reply:
> it is actually

That's the current block on master:
https://github.com/netblue30/firejail/blob/9b1daa6b3abecfa0b270e9202106575cd49aeacc/etc/profile-a-l/librewolf.profile#L38-L53

It does not contain `whitelist /usr/share/pipewire/client.conf` (for example).

> btw will it lead to any problem for example with cachy-browser{-beta,-nightly,etc}?

You mean
- having both installed? No (firecfg does not create symlinks anyway)
- running -beta... with this profile? No (except if paths a different or you use private-bin)

Reply:
no. I meant that you do `include cachy-browser.profile` in cachy-browser-beta(for example).

Reply:
You can do it just like firefox or librewolf handle nightly/beta/dev/esr versions.

Reply:
ah ok. just wanted to be sure

Reply:
how to determine string for `dbus-user.own`? Where is it located?

Reply:
oh nevermind. I did find it

Reply:
Without source-code digging: Run cachy without dbus filtering and look at
- d-feet (GTK)
- qdbusviewer (QT)
- `busctl --user list` (CLI)
- or call ListNames
  - `dbus-send --print-reply --dest=org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus.ListNames`
  - `gdbus call --session --dest org.freedesktop.DBus --object-path /org/freedesktop/DBus --method org.freedesktop.DBus.ListNames`
  - …

Reply:
yeah thx I already did it

Reply:
btw It should be the issue as firefox-developer-edition or librewolf-nightly differ from the parent

for example for librewolf-nightly:
should be `dbus-user.own org.mozilla.librewolfnightly.*` instead of `dbus-user.own org.mozilla.librewolf.*`